package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@sketch7/signalr-orleans",
-  "version": "5.0.0",
+  "version": "5.1.0",
```

```suggestion
  "version": "5.0.0",
```
as its still not published to stable so no need to bump